addons/hive.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: install hive
+  hosts: frontendnodes
+  become: yes
+  roles:
+    - role: java
+    - role: mysql
+      when: HIVE_METASTORE == 'mysql'
```

Comment:
great, as long as it works ;-)

> On May 3, 2016, at 8:22 PM, kjaugustin <notifications@github.com> wrote:
> 
> In addons/hive.yml <https://github.com/futuresystems/big-data-stack/pull/67#discussion_r61977850>:
> 
> > @@ -0,0 +1,11 @@
> > +---
> > +- name: install hive
> > +  hosts: frontendnodes
> > +  become: yes
> > +  roles:
> > +    - role: java
> > +    - role: mysql
> > +      when: HIVE_METASTORE == 'mysql'
> I wrote this with an option to extend this code to provide support for PostgreSQL in the future if needed. At this point the HIVE_METASTORE variable is set using group_vars. Once we add the support for other databases, a user can change the value of this variable to call the corresponding module to provision the metastore(either MySQL or PostgreSQL or Oracle etc.). So shall I keep it the way it is now?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/futuresystems/big-data-stack/pull/67/files/f4c4ba1fd1dd4bb9e51a155204ffa0d92f7121cb#r61977850>